src/app/@shared/card-listing/card-listing.component.scss

```diff
@@ -6,25 +6,61 @@
     object-fit: cover;
 }
 
-.price {
-    color: nb-theme(color-secondary-600);
-}
+// .price {
+//     color: nb-theme(color-secondary-600);
+// }
+
+// .listing-card {
+//     position: relative;
+//     background: white;
+
+//     .card-body {
+//         padding: 15px;
+//     }
+
+//     &__card {
+//         margin-bottom: 0;
+//     }
+//     &__header {
+//         position: relative;
+//         padding: 0;
+//         line-height: 0;
+//     }
+//     &__footer {
+//         display: flex;
+//         justify-content: flex-end;
+//         padding: 5px;
+//     }
+// }
 
 .listing-card {
     position: relative;
-    box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 10px;
-
-    &__card {
-        margin-bottom: 0;
-    }
-    &__header {
-        position: relative;
-        padding: 0;
-        line-height: 0;
-    }
-    &__footer {
-        display: flex;
-        justify-content: flex-end;
-        padding: 5px;
-    }
+    background: white;
+    height: 100%;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.product-actions {
+    margin-top: auto;
+    display: flex;
+    width: 100%;
+    justify-content: flex-end;
+}
```

Comment:
use BEM and sass nested elements.
eg
```
.product {
    &-actions {...}
    &-title {...}
    &-price {...}
}
```
etc

Reply:
I agree with using BEM, but not so much with having selectors as nested elements with & - it makes it harder to search for CSS selector when trying to debug. We can use BEM, but have selectors separated as .product, .product-actions, .product-title.